### PR TITLE
Adds sounds to faxes and Prayer. Different sounds.

### PR DESCRIPTION
**Pull request by Kurfursten**
Also it halves the fax time, because 3 minutes is just too long.

### CODE REVIEW

**Comment by Boxe** [comment]
i'm curious of what the sounds are, do you have links to them?

File: html/changelogs/Kurfursten1911.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 author: Kurfursten
-changes: []
+changes: [Adds special sounds for prayers and faxes to admins -- hopefully you'll get noticed now! Also faxes cooldown faster.]
```
Comment (ComicIronic) [inline html/changelogs/Kurfursten1911.yml]: I do not believe this is going to work.
Try making a new changelog file for this PR, and follow the example.yml formatting.

**Comment by Kurfursten** [comment]
@ComicIronic It works, that's how I've always done mine. The real mistake is that I keep naming all my PR changelogs Kurfursten which doesn't work when you have multiple PR changelogs at once

@Boxe You can view it by clicking "view raw" on the file. I'll look into merging the sound mute into the same toggle as the adminhelp sound mute in the next push

**Comment by Boxe** [comment]
i would advise against grouping it with the ahelp mute, perhaps as its own toggle?

**Comment by Kurfursten** [comment]
@Boxe I have no attachment to the sounds provided, since they're just the sounds that the Issue author requested. Talk to @Probe1

**Comment by Boxe** [comment]
my suggestion is to tie both faxes & prayers to bikehorn.ogg

badmins won't care if they're both tied to the same sound since they're pretty much the same thing

also bikehorn.ogg ties in with BWOINK

**Comment by Probe1** [comment]
@Boxe I could have superyooper halve the volume and duration and @clusterfack said adding a verb to silence only prayer/fax as easy to do.

**Comment by Boxe** [comment]
@Probe1 a different sound entirely would be best i believe, something very short and quieter than adminhelp.ogg since prayers can come in extremely quickly at times & are common in comparison to ahelps

post the results of the changes to the sounds if you perform them please

also a misc request: give the chaplain a special little icon for his pray notification

**Comment by NigglyWiggly** [comment]
>also a misc request: give the chaplain a special little icon for his pray notification

This is a great idea; that way we know specifically which prayers to never respond to.

**Comment by Probe1** [comment]
New fax ogg which is solid
http://www.mediafire.com/listen/2rxfz92j8433iva/fax1.ogg

Two potential player sounds that need feedback
Prayer1
http://www.mediafire.com/listen/zv5o0y0zvqilv56/prayer.ogg
Prayer2
http://www.mediafire.com/listen/v5pnq1aelxehh13/prayer2.ogg

**Comment by Boxe** [comment]
fax sounds is good

player sound 2 is good

**Comment by Rei1226** [comment]
Feel like it needs to be more attentive like how BWOINK is, but honestly anything at this point is good.

**Comment by Boxe** [comment]
i think ahelps are much more worthy of attention in comparison to prayers/faxes, which is why i dislike anything too loud/overbearing.

**Comment by Probe1** [comment]
that's how I knew the fax was what I wanted when I was throwing different sounds together to make it.  it's got a beep at the beginning that wakes you up so you're listening, then an old printers printing noise with a little bit of office noise mixed in.

prayer sucks though, I'm still working on that.  The two I uploaded aren't good enough but come tomorrow I'll have that worked out.

**Comment by Probe1** [comment]
alright final answer

fax - http://www.mediafire.com/listen/2rxfz92j8433iva/fax1.ogg
little beep at the beginning and a printer noise very generic but catchy since the IAA's reliant on them

prayer - http://www.mediafire.com/listen/ciq1lxlu7az1khe/prayer6.ogg
shorter, a little staccato but quieter because prayers will come in at a higher volume.

**Comment by Rei1226** [comment]
I like the fax, the prayer i'm still not sure about.

What about a harp, similar to this?

http://www.freesound.org/people/Soughtaftersounds/sounds/145448/

**Comment by SkowronX** [comment]
Hallelujah

**Pull request merged**